packages/test-utils/dist/vue-test-utils.js

```diff
@@ -17,7 +17,10 @@ function warn (msg) {
 
 var camelizeRE = /-(\w)/g;
 var camelize = function (str) {
-  var camelizedStr = str.replace(camelizeRE, function (_, c) { return c ? c.toUpperCase() : ''; });
```

Comment:
These changes are not related to the issue, files are automatically updated on yarn test or something, but they shouldn't be included in the PR